Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -1231,30 +1233,41 @@ internal async static Task CopyObject_Test2(MinioClient minio)
                                                                 .WithObject(destObjectName);
 
                     await minio.CopyObjectAsync(copyObjectArgs);
-
                 }
                 catch (MinioException ex)
                 {
-                    Console.WriteLine(ex.Message);
-                    StringAssert.Equals(ex.Message, "MinIO API responded with message=At least one of the pre-conditions you specified did not hold");
+                    Assert.IsTrue(ex.Message.Contains("MinIO API responded with message=At least one of the pre-conditions you specified did not hold"));
                 }
 
-                RemoveObjectArgs rmArgs = new RemoveObjectArgs()
+                RemoveObjectArgs rmArgs1 = new RemoveObjectArgs()
```

Comment:
do we not have `RemoveObjects`, why are you making multiple calls?

Reply:
`RemoveObjectsArgs` is implemented for different objects in 1 bucket. In this case it is 2 objects in different buckets.